cmake/cmakelibs.cmake

```diff
@@ -0,0 +1,178 @@
+###################################################
```

Comment:
We have not been 100% consistent with our cmake file headers. But do you mind swapping this for:
```cmake
# Copyright (c) 2014-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed in accordance with the terms specified in
# the LICENSE file found in the root directory of this source tree.

```